core/tests/protractor_utils/waitFor.js

```diff
@@ -120,6 +120,14 @@ var newTabToBeCreated = async function(errorMessage, urlToMatch) {
   }, DEFAULT_WAIT_TIME_MSECS, errorMessage);
 };
 
+/**
+ * @param {string} url - URL to redirect
+ */
+var urlRedirection = async function(url) {
+  // Checks that the current URL matches the expected text.
+  await browser.wait(until.urlIs(url), 15000, 'URL redirection took too long');
```

Comment:
Does the `DEFAULT_WAIT_TIME_MSECS` not work here? Why?

Reply:
It does work actually. I have changed it